ui/src/shared/components/lastRunTaskStatus/LastRunTaskStatus.tsx

```diff
@@ -0,0 +1,80 @@
+// Libraries
+import React, {FC, useRef, useState} from 'react'
+import classnames from 'classnames'
+
+// Components
+import {
+  Icon,
+  IconFont,
+  Popover,
+  Appearance,
+  PopoverInteraction,
+  ComponentColor,
+} from '@influxdata/clockface'
+
+// Styles
+import './LastRunTaskStatus.scss'
+
+interface PassedProps {
+  lastRunError?: string
+  lastRunStatus: string
+}
+
+const LastRunTaskStatus: FC<PassedProps> = ({lastRunError, lastRunStatus}) => {
+  const triggerRef = useRef<HTMLDivElement>(null)
+  const [highlight, setHighlight] = useState<boolean>(false)
+
+  let color = ComponentColor.Success
+  let icon = IconFont.Checkmark
+  let text = 'Task ran successfully!'
+
+  if (lastRunStatus === 'failed' || lastRunError !== undefined) {
```

Comment:
```ts
lastRunError !== undefined
```
is this this check we want here (rather than a more greedy falsy check)?  `lastRunError = false | null | '' | {}` will pass this check and make the app enter an error state. is that desirable?

Reply:
That's a great point and definitely something to consider. My thinking here was that since the `lastRunError` is an optional parameter (it can be undefined), and since the `lastRunError` should be a string if it is defined, the check for explicitly not undefined (meaning a string should exist) should be sufficient. Anything besides a string should trigger an error for us since our API would be behaving unexpectedly if it did return a more generally false check (like `false` `null`). In this case, an empty string being passed in would be more indicative of improper error handling, but not a reason to prevent the functionality